nilearn/input_data/nifti_labels_masker.py

```diff
@@ -205,51 +251,34 @@ def transform(self, imgs, confounds=None):
         """
         self._check_fitted()
 
-        logger.log("loading images: %s" %
-                   _utils._repr_niimgs(imgs)[:200], verbose=self.verbose)
-        imgs = _utils.check_niimg_4d(imgs)
+        # We handle the resampling of labels separately because the affine of
+        # the labels image should not impact the extraction of the signal.
 
+        if not hasattr(self, '_resampled_labels_img_'):
```

Comment:
Just curious, where is _resampled_labels_img_ likely to be set prior to this statement?